### PR TITLE
fix: sticky section at the end of the mobile view in edit and create list

### PR DESCRIPTION
**Pull request by krishnamohanignitiv**

### CODE REVIEW

File: components/b2b/Lists/CreateList/CreateList.tsx [hunk 1]
```diff
@@ -192,17 +192,29 @@ const CreateList = (props: CreateListProps) => {
...
+        )}
         {!mdScreen && (
           <>
-            <Box sx={styles.mobileSaveWindow} gap={2}>
+            <Box
+              display={'flex'}
+              flexDirection={'column'}
+              gap={2}
+              marginTop={'20px'}
+              marginBottom={'-70px'}
```
Comment (kibo-chandradeeptalaha) [inline components/b2b/Lists/CreateList/CreateList.tsx]: don't apply negative margin. Try to make the parent flex and apply gap.

File: components/b2b/Lists/EditList/EditList.tsx [hunk 2]
```diff
@@ -196,20 +195,32 @@ const EditList = (props: EditListProps) => {
...
+      )}
+      {!mdScreen && (
         <>
-          <Grid item xs={12} sx={styles.mobileSaveWindow} gap={2}>
+          <Box
+            display={'flex'}
+            flexDirection={'column'}
+            gap={2}
+            marginTop={'20px'}
+            marginBottom={'-70px'}
```
Comment (kibo-chandradeeptalaha) [inline components/b2b/Lists/EditList/EditList.tsx]: Follow the previous comment

**Review by kibo-chandradeeptalaha — CHANGES_REQUESTED** [review]
Please check

File: components/b2b/Lists/CreateList/CreateList.tsx (hunk 1 above)
Comment (krishnamohanignitiv) [reply components/b2b/Lists/CreateList/CreateList.tsx]: I just checked it. The gap at bottom is coming from footer, so removing the negative margin

**Review by kibo-chandradeeptalaha — APPROVED** [review]

**Pull request merged**